### PR TITLE
Refr/#14 website apiv2

### PR DESCRIPTION
**Pull request by vegarab**
Upgrades website to use version 2.0 of the API.

### CODE REVIEW

File: chatbot/api/v2/test/test_v2_api.py [hunk 1]
```diff
@@ -49,26 +49,28 @@ def test_get_conflict_ids(client):
         response = client.get('/v2/conflict_ids/')
         response_data = json.loads(response.data.decode())
 
+        print(response_data)
```
Comment (torjusti) [inline chatbot/api/v2/test/test_v2_api.py]: stray print?

File: chatbot/web/src/components/DocumentView.js [hunk 2]
```diff
@@ -76,8 +76,10 @@ class DocumentView extends React.Component {
     if (this.state.manual) {
       this.setState(prevState => ({
         manual: {
-          ...prevState.manual,
-          texts: [...prevState.manual.texts, ''],
+		  content: {
+            ...prevState.manual.content,
+            texts: [...prevState.manual.content.texts, ''],
```
Comment (torjusti) [inline chatbot/web/src/components/DocumentView.js]: indentation seems a little wild here

Comment (torjusti) [reply chatbot/web/src/components/DocumentView.js]: (Seems you've mixed tabs and spaces a bit.)

File: chatbot/web/src/components/Search.js [hunk 3]
```diff
@@ -18,8 +18,9 @@ export default class SearchBar extends React.Component {
 
   handleSearch = async url => {
     // Title and ID based on URL.
-    const content = await fetchData(
-      process.env.REACT_APP_SERVER_URL + 'v1/web/docs/?url=' + url,
+	console.log(url)
```
Comment (torjusti) [inline chatbot/web/src/components/Search.js]: This might also not be wanted?

File: chatbot/web/src/utils/Util.js [hunk 4]
```diff
@@ -9,3 +9,14 @@ export async function fetchData(url, method, body) {
...
   return await response.json();
 }
+
+export async function fetchDataWithoutTypeHeader(url, method) {
+  const response = await fetch(url, {
+    method: method,
+	  headers: {
+		  Accept: 'application/json',
+		},
+		body: undefined && JSON.stringify(undefined),
```
Comment (torjusti) [inline chatbot/web/src/utils/Util.js]: You can just omit this line entirely. `undefined && JSON.stringify(undefined)` is always going to evaluate to `undefined`, it seems.

File: chatbot/api/v2/test/test_v2_api.py (hunk 1 above)
Comment (vegarab) [reply chatbot/api/v2/test/test_v2_api.py]: Thanks

File: chatbot/web/src/components/DocumentView.js (hunk 2 above)
Comment (vegarab) [reply chatbot/web/src/components/DocumentView.js]: Indentation is all over the place in all the `.js` files. I'll see if I can apply an automatic, global fix

File: chatbot/web/src/components/Search.js (hunk 3 above)
Comment (vegarab) [reply chatbot/web/src/components/Search.js]: :+1:

File: chatbot/web/src/utils/Util.js (hunk 4 above)
Comment (vegarab) [reply chatbot/web/src/utils/Util.js]: Nice

**Pull request merged**